mail_preview_base/static/src/js/preview.js

```diff
@@ -0,0 +1,160 @@
+/** ********************************************************************************
+    Copyright 2020 Creu Blanca
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+ **********************************************************************************/
+
+odoo.define("mail_preview_base.preview", function (require) {
+    "use strict";
+
+    var DocumentViewer = require("mail.DocumentViewer");
+    var basic_fields = require("web.basic_fields");
+    var registry = require("web.field_registry");
+    var AttachmentBox = require("mail.AttachmentBox");
```

Comment:
This does not exist on 14.0 it has been replaced by https://github.com/odoo/odoo/blob/14.0/addons/mail/static/src/components/attachment_box/attachment_box.js